### PR TITLE
Post-footer home cleanup. Improve action button and CTA.

### PR DESCRIPTION
**Pull request by brandonfancher**
The Home page was looking a bit bare after moving links down into the footer. (heck, it was bare even before that!) This splits the information up a bit...makes it feel more full and easier on the eyes. I think?

Let me know what you think. (Compare eden-dev.vercel.app with this branch's deploy.)

I'm also improving the way `ActionButton` handles external links in this PR...similar to prior improvements to `Link`.

### CODE REVIEW

File: packages/webapp/src/_app/ui/action-button.tsx [hunk 1]
```diff
@@ -56,22 +60,37 @@ export const ActionButton = ({
...
-            className={buttonClass}
-            disabled={disabled}
-        >
-            {children}
-        </button>
+        <NextLink href={href}>
+            <a className={buttonClass} {...targetProps}>
+                {children}
+            </a>
+        </NextLink>
```
Comment (sparkplug0025) [inline packages/webapp/src/_app/ui/action-button.tsx:94]: I think this change will break submit buttons (fill all the fields and press enter). Also I don't think it's a good idea to replace the button semantics with a link inside forms.

Comment (sparkplug0025) [reply packages/webapp/src/_app/ui/action-button.tsx:94]: ohh nevermind... I just saw the button being returned above!

**Review by sparkplug0025 — APPROVED** [review]

File: packages/webapp/src/_app/ui/action-button.tsx (hunk 1 above)
Comment (sparkplug0025) [reply packages/webapp/src/_app/ui/action-button.tsx:94]: nevermind again lol... the button will be reeturned only if onclick is present, but submit buttons don't always have it: https://github.com/eoscommunity/Eden/blob/main/packages/webapp/src/inductions/components/induction-profile-form.tsx#L245

**Review by sparkplug0025 — DISMISSED** [review]
See the comment about Submit buttons

File: packages/webapp/src/_app/ui/action-button.tsx (hunk 1 above)
Comment (brandonfancher) [reply packages/webapp/src/_app/ui/action-button.tsx:94]: yeah, good catch!

**Pull request merged**